app/src/services/cpk.ts

```diff
@@ -1,3 +1,4 @@
+import { txs } from '@gnosis.pm/safe-apps-sdk/dist/txs'
```

Comment:
What do you think about creating a separate service file for safe transactions instead of modifying the cpk service?

Reply:
Open to it for sure but not sure what the benefit would be?

Reply:
Yeah, now looking at the code again I don't think it's quite necessary